src/components/scene/inspector.js

```diff
@@ -1,8 +1,8 @@
 /* global AFRAME, INSPECTOR_VERSION */
-var AFRAME_INJECTED = require('../../constants').AFRAME_INJECTED;
-var pkg = require('../../../package');
-var registerComponent = require('../../core/component').registerComponent;
-var utils = require('../../utils/');
+import { AFRAME_INJECTED } from '../../constants/index.js';
+import { registerComponent } from '../../core/component.js';
+import pkg from '../../../package.json';
```

Comment:
Worth pointing out that importing JSON files would also not work in a no-build setup. Perhaps once JSON modules are readily available in browsers.

Reply:
The new web standard will be:
```js
import pkg from '../../../package.json' with { type: "json" };
```
currently supported by node if you test it with `npm run test:node`.
For webpack, babel-generator gives a warning 
```
You are using import attributes, without specifying the desired output syntax.
Please specify the "importAttributesKeyword" generator option, whose value can be one of:
 - "with"        : `import { a } from "b" with { type: "json" };`
 - "assert"      : `import { a } from "b" assert { type: "json" };`
 - "with-legacy" : `import { a } from "b" with type: "json";`
```
Not sure where you configure it.

Reply:
For me webpack (or rather babel) complains that the `@babel/plugin-syntax-import-attributes` is missing, and after installing and adding it to the plugins of `babel-loader`, it works fine.

But I don't think there's any upside to pre-emptively using this syntax. In fact, I'd rather see us avoiding importing `package.json` altogether. Currently the entirety of `package.json` gets embedded into the A-Frame bundle, despite only needing the version property.

Reply:
I agree with you, I wouldn't introduce the new syntax either. We should just inject the aframe version as part of the build like we do for INSPECTOR_VERSION really. Importing package.json was only to get the aframe version when importing from node AFAICT. I don't see a real use of importing aframe in node where there is no custom elements api.

Reply:
It was used for logging the THREE and WebVR polyfill versions in the past as well, but the former now uses `THREE.REVISION` and the latter has since been removed. Injecting the version is a lot cleaner.